website/docs/r/schema_site_vrf_region_cidr_subnet.html.markdown

```diff
@@ -35,7 +35,7 @@ resource "mso_schema_site_vrf_region_cidr_subnet" "sub1" {
 * `template_name` - (Required) Template where Vrf Region Cidr Subnet to be created.
 * `cidr_ip` - (Required) The IP range of for the region CIDR where Vrf Region Cidr Subnet to be created.
 * `ip` - (Required) The IP subnet of this region CIDR.
-* `zone` - (Required) The name of the zone for the region CIDR subnet.
+* `zone` - (Optional) The name of the zone for the region CIDR subnet.
```

Comment:
Do we need to make it more detailed like "`zone` - (Optional) The name of the zone for the region CIDR subnet and only required for AWS sites"

Reply:
Fixed